aws/resource_aws_route53_record_test.go

```diff
@@ -2074,115 +2110,119 @@ data "aws_availability_zones" "available" {
 }
 
 resource "aws_route53_zone" "main" {
-	name = "notexample.com"
+  name = "notexample.com"
 }
 
 resource "aws_elb" "alias_change" {
-  name = "foobar-tf-elb-alias-change"
+  name               = "foobar-tf-elb-alias-change"
   availability_zones = slice(data.aws_availability_zones.available.names, 0, 1)
 
   listener {
-    instance_port = 80
+    instance_port     = 80
     instance_protocol = "http"
-    lb_port = 80
-    lb_protocol = "http"
+    lb_port           = 80
+    lb_protocol       = "http"
   }
 }
 
 resource "aws_route53_record" "elb_alias_change" {
-  zone_id = "${aws_route53_zone.main.zone_id}"
-  name = "alias-change"
-  type = "A"
+  zone_id = aws_route53_zone.main.zone_id
+  name    = "alias-change"
+  type    = "A"
 
   alias {
-    zone_id = "${aws_elb.alias_change.zone_id}"
-    name = "${aws_elb.alias_change.dns_name}"
+    zone_id                = aws_elb.alias_change.zone_id
+    name                   = aws_elb.alias_change.dns_name
     evaluate_target_health = true
   }
 }
 `
 
 const testAccRoute53RecordAliasChangePost = `
 resource "aws_route53_zone" "main" {
-	name = "notexample.com"
+  name = "notexample.com"
 }
 
 resource "aws_route53_record" "elb_alias_change" {
-  zone_id = "${aws_route53_zone.main.zone_id}"
-  name = "alias-change"
-  type = "CNAME"
-  ttl = "30"
+  zone_id = aws_route53_zone.main.zone_id
+  name    = "alias-change"
+  type    = "CNAME"
+  ttl     = "30"
   records = ["www.terraform.io"]
 }
 `
 
 const testAccRoute53RecordConfigEmptyName = `
 resource "aws_route53_zone" "main" {
-	name = "notexample.com"
+  name = "notexample.com"
 }
 
 resource "aws_route53_record" "empty" {
-	zone_id = "${aws_route53_zone.main.zone_id}"
-	name = ""
-	type = "A"
-	ttl = "30"
-	records = ["127.0.0.1"]
+  zone_id = aws_route53_zone.main.zone_id
+  name    = ""
+  type    = "A"
+  ttl     = "30"
+  records = ["127.0.0.1"]
 }
 `
 
 const testAccRoute53RecordConfigLongTxtRecord = `
 resource "aws_route53_zone" "main" {
-	name = "notexample.com"
+  name = "notexample.com"
 }
 
 resource "aws_route53_record" "long_txt" {
-    zone_id = "${aws_route53_zone.main.zone_id}"
-    name = "google.notexample.com"
-    type = "TXT"
-    ttl = "30"
-    records = [
-        "v=DKIM1; k=rsa; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAiajKNMp\" \"/A12roF4p3MBm9QxQu6GDsBlWUWFx8EaS8TCo3Qe8Cj0kTag1JMjzCC1s6oM0a43JhO6mp6z/"
-    ]
+  zone_id = aws_route53_zone.main.zone_id
+  name    = "google.notexample.com"
+  type    = "TXT"
+  ttl     = "30"
+  records = [
+
+    <<EOF
+v=DKIM1; k=rsa; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAiajKNMp" "/A12roF4p3MBm9QxQu6GDsBlWUWFx8EaS8TCo3Qe8Cj0kTag1JMjzCC1s6oM0a43JhO6mp6z/
+EOF
+
+  ]
```

Comment:
This could be left as a regular string, and not a HEREDOC

Reply:
:blergh: